CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+
+#### How Do I Submit a Good Bug Report?
+
+> You must never report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead follow the instructions outlined by our [Security Incident & Coordinated Vulnerability Disclosure](https://help.fullstory.com/hc/en-us/articles/360020624254-Security-Overview#h_01G9QN7Y3GYW36M01HG1RRTFXE) process or email [psirt@fullstory.com](mailto:psirt@fullstory.com).
+
+If you run into an issue with the project:
+- Create a new [GitHub issue](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/issues/new?&template=bug_report.md) with our bug report template.
+	- Alternatively you can submit your request through the [FullStory Help Center](https://help.fullstory.com/hc/en-us/requests/new) with details about the error you're running into.
+- Provide the information you collected in the previous section. 
+- For faster remediation you should isolate the problem and create a reduced test case or provide a minimal reproducible example.
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps.
+- If the team is able to reproduce the issue, it will be handled accordingly by our support team. We will do our best to maintain an open line of communication as we work on your request.
+
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion such as a completely new feature or minor improvements to existing functionality. Following these guidelines will help maintainers to understand your suggestion.
+
+- Make sure that you are using the latest version.
+- Find out if the functionality is already covered.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to convince the maintainers of the merits of this feature. We want features that will be useful to the majority of users.
+
+#### Creating a Feature Request
+
+Enhancements are suggested by creating a new [GitHub issue](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/issues/new?&template=feature_request.md) with our feature request template.
+
+- Use a descriptive title for the issue to identify the suggestion.
+- Provide a step-by-step description of the suggested enhancement, include screenshots and animated GIFs if necessary.
+- Describe the current behavior and explain which behavior you expected to see instead. 
+- Describe alternatives to your solution and why the alternatives do not work for you.
+- Explain why this enhancement would be useful.
+
+## Pull Request Process
+
+All pull requests should include a clear list of your changes (read more about [pull requests](http://help.github.com/pull-requests/)). Any pull request that makes functional changes to the library should also include unit tests. We can always use more test coverage. 
```

Comment:
The `pull requests` link seems to redirect to GitHub's base help page.

Reply:
yes, that was the intention; for beginners who might not be familiar with what pull request are. I can remove this section though 🤔 wdyt?

Reply:
Maybe this would be a good link? https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

Reply:
added that one 👍